sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -338,7 +338,7 @@ querySpecification
        (RECORDREADER recordReader=STRING)?
        fromClause?
        (WHERE where=booleanExpression)?)
-    | ((kind=SELECT setQuantifier? namedExpressionSeq fromClause?
+    | ((kind=SELECT setQuantifier? namedExpressionSeq (intoClause? fromClause)?
```

Comment:
```sql
SELECT 1 
INTO newtable
```
This won't work because we need oldtable info to create newtable. So the sql should be
```sql
SELECT 1
INTO newtable 
FROM oldtable
```
The result from my test is: a new table called newtable was created, one column called 1 has the length of oldtable.rows.length and all elements are 1.
Did you mean there is no _FROM_?